tests/misc/CachingTest.py

```diff
@@ -122,20 +122,28 @@ def test_caching_results(self):
             retval, stdout, stderr = execute_coala(
                 coala.main,
                 'coala',
+                '--non-interactive', '--no-color',
```

Comment:
No.  Escape sequences are printed in raw bytes when there is an error in a test, and as pytest trims long strings, all the color information is shown while the useful information is not show.  See
https://travis-ci.org/jayvdb/coala/jobs/235260955#L813